.github/workflows/test-binaries.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Tezos binaries
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - PruStephan/Add_tests_for_packages
```

Comment:
Just a reminder so you don't forget to change it

Reply:
Actually ...... thanks, this issue exists for so long, that I forgot, that I even inserted it there